app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<%=render "shared/navbar", right_el: "back", left_el: "none", title: "New Transaction"%>
+<%= form_with model: @transaction, url: category_transactions_path(params[:category_id]), method: :post, local: true, scope: :transaction, class:"p-3" do |f|%>
+    <div class="mb-3">
+        <%= f.label :name, "Name", class:"form-label" %><br>
+        <%= f.text_field :name,  :required => true, class:"form-control", placeholder: "Enter the name of the transaction" %>
+    </div>
+    <div class="mb-3">
+        <%= f.label :amount, "Amount", class:"form-label" %><br>
+        <%= f.number_field :amount,  :required => true, class:"form-control", placeholder: "How much did the transaction cost" %>
+    </div>
+    <button type="submit" class="btn btn-primary">Create</button>
+<%end%>
+
```

Comment:
- [ ] As stated in the project requirement in the section of "Add a new transaction" page the user should fills out a form to **create a new transaction with: name (mandatory), amount (mandatory) and  categories (mandatory at least one) but In the case of yours the form contain name and amount so Kindly add categories at least one .**
![Capture](https://github.com/Mahamadou-Nouridine/rails-budget-app/assets/93598189/bae1126f-dc49-47dc-9b6a-c7af46569079)